apps/logging/request_logger.py

```diff
@@ -4,35 +4,68 @@
 from django.conf import settings
 from apps.dot_ext.loggers import get_session_auth_flow_trace
 
+CRITICAL = logging.CRITICAL
+FATAL = logging.FATAL
+ERROR = logging.ERROR
+WARNING = logging.WARNING
+WARN = logging.WARN
+INFO = logging.INFO
+DEBUG = logging.DEBUG
+NOTSET = logging.NOTSET
+
+
+def getLogger(name=None):
+    return RequestLogger(None, name) if name else BasicLogger()
```

Comment:
```suggestion
    return RequestLogger(request, name) if request else BasicLogger()
```

Reply:
> 

some times at the point of logging, there is no request object, e.g. some logging function is called by signal, some signals are hooked with django model operations - table record insert / delete, etc. 

some times, e.g. SLS loggers, the request is not the request going through loggingmiddleware, the requests are on a new session between BB2 vs SLS...

Reply:
That might be the case - but we should still allow for the option. Otherwise this function is ALWAYS returning a BasicLogger. A RequestLogger without a request object passed in, is really just a BasicLogger (since that is the parent).

Reply:
Also worth noting - that if we don't want to use the RequestLogger, we are basically losing out on the log tracing capabilities we are adding with this story. So basically if we can't use the RequestLogger most of the time, the value of this work is greatly diminished.

Reply:
Good to have multiple constructors to accommodate use cases...